test/functional/NativeTest/playlist.xml

```diff
@@ -191,6 +191,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</subsets>
 	</test>
 	<test>
+		<aot>explicit</aot>
```

Comment:
For the explicit case, the command will be run as is. Make sure you have aot options and the test will be executed in multiple iterations if needed.

Reply:
These are SCC tests, so that's why I marked them as `explicit`.